docs/content/staking/index.mdx

```diff
@@ -8,28 +8,24 @@ This document provides an introduction to staking FLOW tokens on the Flow networ
 Staking is an important part of the security of a proof-of-stake (PoS) blockchain.
 Running nodes and staking tokens contributes to this security and is rewarded accordingly.
 
+
 ## What is Staking?
 
-A blockchain is a global network of computers working together
-to maintain the security and integrity of a shared state that applications and people around the world use. 
-Every node operator has a small part of the responsibility to keep the network running smoothly 
-and to ensure that other node operators are doing the same. 
-This large distribution of resources makes it so that control is never centralized 
-under one person or organization.
+Flow is a global network of computers working together
+to maintain the security and integrity of it's user's data.
+
+This global network is made up of many individual nodes. Every node in the network shares a small part of the responsibility to keep the network running smoothly 
+and to ensure that other nodes are doing the same. This shared responsibility is sometimes called decentralization, because no single node is responsible for th e security and integrity of the network and the data it contains.
```

Comment:
```suggestion
and to ensure that other nodes are doing the same. This shared responsibility is sometimes called decentralization, because no single node is responsible for the security and integrity of the network and the data it contains.
```